pkg/collector/corechecks/cluster/ksm/kubernetes_state.go

```diff
@@ -745,17 +744,24 @@ func ownerTags(kind, name string) []string {
 	return tags
 }
 
-// podLabelsMapperOverride defines the label mapper overrides
-// that should be applied on pod metrics only.
-var podLabelsMapperOverride = map[string]string{"phase": "pod_phase"}
-
 // labelsMapperOverride allows overriding the default label mapping for
 // a given metric depending on the metric family.
-// Current use-case:
+// Current use-cases:
 //   - `phase` tag should be mapped to `pod_phase` on pod metrics only.
+//   - Ingress metrics have generic tag names (host/path/service_name/service_port).
+//     It's important to have them in a dedicated mapper override for ingresses.
 func labelsMapperOverride(metricName string) map[string]string {
 	if strings.HasPrefix(metricName, "kube_pod") {
-		return podLabelsMapperOverride
+		return map[string]string{"phase": "pod_phase"}
+	}
+
+	if strings.HasPrefix(metricName, "kube_ingress") {
+		return map[string]string{
+			"host":         "ingress_host",
```

Comment:
I was thinking that maybe we can have `kube_` prefix too. like for other `kubernetes_state` tags